static/app/views/performance/traceDetails/utils.tsx

```diff
@@ -19,14 +19,20 @@ export function getTraceDetailsUrl(
   traceSlug: string,
   dateSelection,
   query: Query,
-  timestamp?: string | number,
+  timestamp: string | number | undefined,
   eventId?: string
 ): LocationDescriptorObject {
   const {start, end, statsPeriod} = dateSelection;
 
+  // Give precedence to timestamp over statsPeriod
+  if (timestamp) {
```

Comment:
Do we need this? The trace page will already take care of this btw, we can likely keep it so that we can continue to forward it to other pages

Reply:
Ahh yes, you are right about this. I'll remove this block